projects/canopy/src/styles/blockquote.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Components/Block quote/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Block quote
+
+<p class="standfirst">A quote allows to repeat words that someone else has said or written.</p>
+
+<Story id="components-block-quote-examples--quote"></Story>
+
+## Usage
+
+The [blockquote](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote) and [cite](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite) styles are part of the Canopy global stylesheet, therefore using the HTML below should be enough:
+
+```html
+<blockquote>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+  <cite>The author</cite>
+</blockquote>
+```
+
+In addition to that, the side line's colour can be changed by overriding the `--blockquote-border-left-colour` css variable.
+
+## Research on this pattern
+
+This is based on best practice.
+
+## Help improve this page
```

Comment:
@elenagarrone This section repeats on every file, right? Could we abstract it out into an import that each mdx file consumes?

Reply:
FYI the PR for that change is here: https://github.com/Legal-and-General/canopy/pull/1067